src/DssSpell.sol

```diff
@@ -41,7 +37,25 @@ contract DssSpellAction is DssAction {
     uint256 constant RAY = 10**27;
     uint256 constant RAD = 10**45;
 
-    address constant MCD_FLASH = 0x1EB4CF3A948E7D72A198fe073cCb8C7a948cD853;
+    bytes32 constant ILK_PSM_USDC_A     = "PSM-USDC-A";
+
+    // Growth Core Unit
+    address constant GRO_MULTISIG        = 0x7800C137A645c07132886539217ce192b9F0528e;
+    // Content Production Core Unit
+    address constant MKT_MULTISIG        = 0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60;
+    // GovAlpha Core Unit
+    address constant GOV_MULTISIG        = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
+    // Real-World Finance Core Unit
+    address constant RWF_MULTISIG        = 0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09;
+    // Risk Core Unit
+    address constant RISK_MULTISIG       = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
```

Comment:
nit: this address is an EOA, not a multisig

https://etherscan.io/address/0xd98ef20520048a35EdA9A202137847A62120d2d9

Although I suppose the naming is _really_ important.

Reply:
Changed this to `RISK_CU_EOA`